### DISCUSSION TITLE
get pending reboot hosts

### DISCUSSION THREAD

**Discussion by Cgiancasproideos** [CrowdStrike/psfalcon#346] [General]
Hi,
how would it be possible to extract all hosts in pending reboot state TRUE?

Thanks for support

**Comment by bk-cs** [comment]
If you have Falcon Exposure Management (a.k.a. Falcon Spotlight), you can use the [Installed Patches](https://falcon.crowdstrike.com/spotlight/patches) report to view pending reboot status.

There isn't an API call that will report "reboot pending" directly. You'd either need to create a scheduled search (assuming there's an event that tracks it), or use Real-time Response to run a PowerShell script to check all their computers for reboot status. If you use a scheduled search, you can download that via API after. I don't know off the top of my head if there's a "pending reboot" event, but there probably is, since it's tracked by the _Installed Patches_ report.

**Reply by Cgiancasproideos** [reply]
Thanks for the reply, I decided to use the Installed Patches report
